docs/content/docs/guide/shortcodes/cards.md

```
@@ -0,0 +1,126 @@
+---
+
+## Example
+
+{{< cards >}}
+{{< card link="../callout" title="Callout" icon="warning" >}}
+{{< card link="../callout" title="Card with tag" icon="tag" tag="custom tag">}}
+{{< card link="/" title="No Icon" >}}
+{{< /cards >}}
+
+{{< cards >}}
+{{< card link="/" title="Image Card" image="https://github.com/user-attachments/assets/71b7e3ec-1a8d-4582-b600-5425c6cc0407" subtitle="Internet Image" >}}
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)